rules/rules.libsonnet

```diff
@@ -121,23 +121,23 @@
           {
             record: 'cluster_quantile:%s:histogram_quantile' % metric,
             expr: |||
-              histogram_quantile(%(quantile)s, sum(rate(%(metric)s_microseconds_bucket{%(kubeSchedulerSelector)s}[5m])) without(instance, %(podLabel)s)) / 1e+06
+              histogram_quantile(%(quantile)s, sum(rate(%(metric)s_bucket{%(kubeSchedulerSelector)s}[5m])) without(instance, %(podLabel)s)) / 1e+06
```

Comment:
Aren't the metrics suffixed with `_seconds_bucket`, not just `_bucket`?

I think we can drop the `/ 1e+06` now that the unit is seconds (also on line 140).

Reply:
Yeah unfortunately we realized this here as well, while they are indeed in seconds now, we did forget to append the seconds suffix.